sdcm/nemesis.py

```diff
@@ -5166,8 +5166,6 @@ def _disrupt_toggle_audit(self, store: AuditStore):
             reduce categories by excluding DML and QUERY,
             verify DDL are logged in audit log correctly. Leaves audit log enabled this way.
         """
-        if not self.target_node.is_enterprise:
```

Comment:
I would replace this with version check, if the major version is lower than 6 this is unsupported nemesis